### ISSUE TITLE
Weird image glitch with Universal Rendering Pipeline

### ISSUE THREAD

**Issue by Alvarden15** [googlevr/gvr-unity-sdk#1065]
### ERROR ENCOUNTERED
There is a weird glitch that distorts the image of the app when it's build with the Universal Rendering Pipeline.

### DETAILS
- Unity: 2019.3.7b
- Google VR SDK for Unity: v1.200.1
- Target OS: Android

### STEPS TO REPRODUCE THE ISSUE
 1. Download the URP from the Package Manager
 2. Install and update the project's materials
 3. Go to the Project settings and set the Stereo Rendering Mode to single pass.
 4. Enable Post processing to the camera.
 5.  Build and run the app
 6. Install the app
 7. Run the app on the phone

Here's the result:
![Screenshot_20191023-103100_Puruchuco](https://user-images.githubusercontent.com/31715989/67411783-c5d37000-f583-11e9-866d-26105aea59f4.jpg)

Can't confirm if it's something from the Pipeline, the SDK or the editor. If anyone has an answer, please let me know.

### ADDITIONAL COMMENTS
I tested it in a Samsung Galaxy S7. Not sure if it also happens to other devices.
Here are my Project settings on detail:

![ProjectSettings1](https://user-images.githubusercontent.com/31715989/67411930-03d09400-f584-11e9-82a6-d47fdf5747a7.PNG)

![ProjectSettings2](https://user-images.githubusercontent.com/31715989/67411938-07641b00-f584-11e9-90ab-33d7b58ca1df.PNG)

![ProjectSettings3](https://user-images.githubusercontent.com/31715989/67411941-09c67500-f584-11e9-836b-688f68ed90b3.PNG)

Yes, i lowered some graphic aspects on purpose to reduce heating.

**Comment by zxcvb7653** [comment]
Using:
Google VR package
VFX Graph
Unity 2019.3

--------------------------------

I HAVE THE SAME EXACT ISSUES! and this issue applies to IOS as well in unity 2019.3beta

So if you look at the images below.
There are a lot of artefacts when building to android using both Vulkan and OpenGLES3 if I use Single-pass rendering mode for VR.

I have tried Building only Vulkan or only OpenGLES3 they both result in artefact with Single-pass.


Vulkan with Single-Pass
![image](https://user-images.githubusercontent.com/39035665/67864775-174ca380-fb1e-11e9-9c30-92516a3e2503.png)


Multi-pass with OpenGLES3. (OpenGLES3 doesn't seem to work with VFX graph on android, I have tried a build without VR and it will still cause Artefacts)
![image](https://user-images.githubusercontent.com/39035665/67864827-29c6dd00-fb1e-11e9-8825-e99612854a37.png)


The correct scene view: The fix for the artefacts is to use Multi-pass with Vulkan, but then it would no longer be stereoscopic VR
![image](https://user-images.githubusercontent.com/39035665/67864846-377c6280-fb1e-11e9-89dd-96bf65518e1f.png)


But if I using single-pass it wouldn't let me use Vulkan Graphics API for VR.
![image](https://user-images.githubusercontent.com/39035665/67864870-406d3400-fb1e-11e9-9fac-b013520f4530.png)


So I am in a catch 22 situation. Please, can any provide a suitable solution that will allow me to render in stereoscopic view and in Vulkan?

**Comment by andrewio** [comment]
Have similar issues, but not doubling pieces. I have black gaps instead. If I disable Post processing stack v2, then all is OK with OpenGLES3

**Comment by Alvarden15** [comment]
Yeah, apparently Google VR and Post Processing Stack don't get along. I tested it too and the black gaps appeared, obstructing the view.

**Comment by zxcvb7653** [comment]
Oh is this to do with "Volume" Post-processing? So if I remove it, it would look normal?

**Comment by andrewio** [comment]
> Oh is this to do with "Volume" Post-processing? So if I remove it, it would look normal?

I didn't test post processing stack v2 with separate area (as v2 can) instead of whole camera view, but please test it and tell here it's working or not.
